core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -604,6 +604,14 @@ class CivilizationInfo {
 
     fun isAtWar() = diplomacy.values.any { it.diplomaticStatus == DiplomaticStatus.War && !it.otherCiv().isDefeated() }
 
+    fun hasOpenBordersTo(otherCiv: CivilizationInfo): Boolean {
+        if (otherCiv == this) return true // own borders are always open
+        if (otherCiv.isBarbarian() || isBarbarian()) return true // barbarians do not care of borders
```

Comment:
Preparing for the day Barbarians can actually _have_ cities? OK... But I'd say you wouldn't have the equivalent of open borders with them... Meaning if barbs ever took ownership of a tile on your road, it should count as disrupted

Reply:
Good thoughts, probably, their territory should be always closed for the trades. Will change that, thank you.